localstack/services/iam/iam_starter.py

```diff
@@ -255,6 +255,19 @@ def inline_policy_unapply_policy(self, backend):
 
     InlinePolicy.unapply_policy = inline_policy_unapply_policy
 
+    def instance_profile_create_from_cloudformation(resource_physical_name, cloudformation_json, region_name):
+        properties = cloudformation_json['Properties']
+
+        role_ids = [role.id for role in moto_iam_backend.roles.values() if role.name in properties['Roles']]
+
+        return moto_iam_backend.create_instance_profile(
+            name=resource_physical_name,
+            path=properties.get('Path', '/'),
+            role_ids=role_ids,
+        )
+
+    InstanceProfile.create_from_cloudformation_json = instance_profile_create_from_cloudformation
```

Comment:
Looks great, thanks for this PR @duongpv7 👍 

In a future iteration, let's better use the new CF approach and remove the dependency to `moto` here.. Thanks